lobby.html

```diff
@@ -310,38 +320,54 @@
     }
 
     function clearCanvas() {
-        socket.send(JSON.stringify({Type: "clear-drawing-board"}));
+        socket.send(JSON.stringify({
+            Type: "clear-drawing-board"
+        }));
     }
 
     const sendMessage = () => {
-        socket.send(JSON.stringify({Type: "message", Data: messageInput.value}));
+        socket.send(JSON.stringify({
+            Type: "message",
+            Data: messageInput.value
+        }));
         messageInput.value = "";
 
         // Necessary in order to keep the page from submitting.
         return false;
     }
 
     function chooseWord(index) {
-        socket.send(JSON.stringify({Type: "choose-word", Data: index}));
+        socket.send(JSON.stringify({
+            Type: "choose-word",
+            Data: index
+        }));
         wordDialog.style.visibility = "hidden";
     }
 
     function onClickKickButton(playerId) {
-        socket.send(JSON.stringify({Type: "kick-vote", Data: playerId}));
+        socket.send(JSON.stringify({
+            Type: "kick-vote",
+            Data: playerId
+        }));
     }
 
     new MutationObserver(
         () => {
             messageContainer.scrollTop = messageContainer.scrollHeight;
         }
-    ).observe(messageContainer, {attributes: false, childList: true, subtree: false});
+    ).observe(messageContainer, {
+        attributes: false,
+        childList: true,
+        subtree: false
+    });
 
     socket.onopen = () => {
         console.log("Successfully Connected");
     };
 
     socket.onmessage = event => {
         var parsed = JSON.parse(event.data);
+        console.log("event: " + parsed.Type);
```

Comment:
I'd prefer not unnecessarily having logger code on the client. If you need that for debugging purposes, you can simply add that real quick.